satpy/composites/__init__.py

```diff
@@ -1555,3 +1555,34 @@ def __call__(self, projectables, nonprojectables=None, **info):
 
         masked_projectable = projectable.where(lon_min_max)
         return super(LongitudeMaskingCompositor, self).__call__([masked_projectable], **info)
+
+class MSGQuicklook(GenericCompositor):
+    """Pseudo true color composite.
+        source: https://www.frontiersin.org/articles/10.3389/frsen.2021.666516/full
+
+    Args:
+        ch16_w (float): weight for red channel (1.6 um).
+        ch08_w (float): weight for green channel (0.8 um).
+        ch06_w (float): weight for blue channel (0.6 um).
+    """
+
+    def __call__(self, projectables, *args, **kwargs):
+        """Generate the composite."""
+        projectables = self.match_data_arrays(projectables)
+        ch16 = projectables[0]
+        ch08 = projectables[1]
+        ch06 = projectables[2]
+
+        MIR = 0.001 + 0.888717 * ch16
+        NIR = 0.001 + 0.731068 * ch08
```

Comment:
These are not constants, so should have lower case variable names.

Reply:
ok, lowercase